HW0/P6/P6.txt

```diff
@@ -1 +1,31 @@
-# Your answers here
+By: Matheus C. Fernandes
+
+Count to 10:
+	Example output:
+		Hi Job 0
+		Hi Job 1
+		Hi Job 2
+		Hi Job 3
+		Bye Job 3Bye Job 2
+
+		Bye Job 1
+		Bye Job 0
+		Hi Job 4
+		Hi Job 5
+		Hi Job 6
+		Hi Job 7
+		Bye Job 5
+		Bye Job 4Bye Job 6
+
+		Bye Job 7
+		Hi Job 8
+		Hi Job 9
+		Bye Job 8Bye Job 9
+
+		[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+		[Finished in 1.0s]
+
+	The order in which the jobs are completed change from one run to the other, this is a result of doing the jobs in parallel. This could be a problem when the input of a job requires an output of a previous job or a job that is in the order of the process. Another possible issue when we want the output to be in order of processes. For example, I know I have 4 cores, so I have the dependence of job 5 on one of the 4 first jobs. This scenario could create a conflict.
+
+How much faster?:
```

Comment:
Do you know why your plot seems to shoot above 4 at certain times? That is pretty weird. Naively, the maximum speedup should be 4x, right?